src/kern/npf_alg_pptp.c

```diff
@@ -349,8 +356,7 @@ pptp_gre_get_state(npf_cache_t *npc, pptp_tcp_ctx_t *tcp_ctx,
 	if (gre_state) {
 		gre_state->orig_client_call_id = client_call_id;
 		gre_state->call_id[CLIENT_CALL_ID] = trans_client_call_id;
-		gre_state->call_id[SERVER_CALL_ID] = 0;
```

Comment:
I'd keep this cleared.

Reply:
there is no need for that anymore since the value is used only if the corresponding flag is up

Reply:
It might be defensive and initialise these values for diagnostic purposes, but I am neutral on this.. So, I'll just resolve this issue.